### ISSUE TITLE
LibDwarf path isn't persisting properly in CMake

### ISSUE THREAD

**Issue by mgaudet** [eclipse/omr#1574]
If you configure OMR using CMake, it will work, and successfully build DDR. 

However, if you then touch an arbitrary CMake file, such that CMake self-reconfigures, it then will fail to build omr, as it no longer is able to find libdwarf.

```
Processing tdf file /home/magaudet/open/omr/util/hashtable/hashtable.tdf
Calculating groups for hashtable
Creating header file: /home/magaudet/open/omr/build/util/hashtable/ut_hashtable.h
Creating pdat file: /home/magaudet/open/omr/build/util/hashtable/ut_hashtable.pdat
Creating c file: /home/magaudet/open/omr/build/util/hashtable/ut_hashtable.c
[166/166] Linking CXX executable ddr/tools/ddrgen/omr_ddrgen
magaudet@ubuntu1604VM:~/open/omr/build (detached*)$ touch ../cmake/compiler_support.cmake 
magaudet@ubuntu1604VM:~/open/omr/build (detached*)$ ninja omr_ddrgen
[1/1] Re-running CMake...
-- Starting with CMake version 3.5.1
CMake Warning at CMakeLists.txt:24 (message):
  CMake support is currently experimental.
<snip> 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/magaudet/open/omr/build
[3/10] Building CXX object ddr/lib/ddr-scanner/CMakeFiles/omr_ddr_scanner.dir/dwarf/DwarfScanner.cpp.o
FAILED: /usr/lib/ccache/c++   -DHAVE_DWARF_H=1 -DHAVE_LIBDWARF_LIBDWARF_H=1 -DJ9HAMMER -DLINUX -DUT_DIRECT_TRACE_REGISTRATION -D_FILE_OFFSET_BITS=64 -I. -I.././include -I.././include_core -I.././third_party -I../ddr/include -pthread   -std=gnu++11 -MMD -MT ddr/lib/ddr-scanner/CMakeFiles/omr_ddr_scanner.dir/dwarf/DwarfScanner.cpp.o -MF ddr/lib/ddr-scanner/CMakeFiles/omr_ddr_scanner.dir/dwarf/DwarfScanner.cpp.o.d -o ddr/lib/ddr-scanner/CMakeFiles/omr_ddr_scanner.dir/dwarf/DwarfScanner.cpp.o -c ../ddr/lib/ddr-scanner/dwarf/DwarfScanner.cpp
In file included from ../ddr/lib/ddr-scanner/dwarf/DwarfScanner.cpp:21:0:
../ddr/include/ddr/scanner/dwarf/DwarfScanner.hpp:42:31: fatal error: libdwarf/libdwarf.h: No such file or directory
compilation terminated.
ninja: build stopped: subcommand failed.
magaudet@ubuntu1604VM:~/open/omr/build (detached*)$ 
```

**Comment by rwy0717** [comment]
I'll fix it, thanks for reporting :)

**Comment by mgaudet** [comment]
Closing this, as I believe it was fixed in #1585.
